src/web/src/components/Login/LoggedOut.tsx

```diff
@@ -1,20 +1,11 @@
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
-import config from '../../config';
+
+import useAuth from '../../hooks/use-auth';
 import HeaderButton from '../Header/HeaderButton';
 
 const LoggedOut = () => {
-  const { loginUrl } = config;
-
-  return (
-    <HeaderButton
-      button={{
-        href: loginUrl,
-        title: 'Log In',
-        ariaLabel: 'log in',
-        Icon: ExitToAppIcon,
-      }}
-    />
-  );
+  const { login } = useAuth();
+  return <HeaderButton title="Log In" onClick={() => login()} Icon={ExitToAppIcon} />;
```

Comment:
Can't we just do `onClick={login}`?

Reply:
Huh, this is weird that I didn't get this locally.  Thanks a lot for catching it, @birtony: review works!  I'll make a fix and push again.